client/cz-component-dash.html

```diff
@@ -104,57 +104,74 @@
           type: 'Object',
           value: function() { return []; },
         },
-        component: {
+        updateSLO: {
           type: 'Object',
-          value: function() { return {}; },
+          value: function() { return null; },
         },
-        updateSLO: {
+        issueList: {
           type: 'Object',
-          value: function() { return {}; },
+          value: function() { return null; },
+        },
+        issueBreakdown: {
+          type: 'Object',
+          value: function() { return {}; }
         }
       },
 
+      observers: [
+        'updateIssueBreakdown(issueList)',
+      ],
+
       attached: function() {
-        registerSource('cz-config', 'users', function(users) {
+        registerSource('cz-config', 'users', users => {
```

Comment:
(this is an example of an unrelated change that I don't mind seeing in a review. It's well contained, improves readability, and doesn't force me to chase changes all over the file.)